### PR TITLE
[C] Update vulnerable dependencies

### PR DESCRIPTION
**Pull request by alexgoff**
For JIRA: "EPO-5567 #IN-PROGRESS #comment update package dependencies
with security vulnerabilities"

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -83,10 +66,33 @@
     "sass": "^1.26.12",
     "striptags": "^3.1.1",
     "styled-components": "^5.2.0",
-    "stylelint": "^13.8.0",
-    "stylelint-config-recess-order": "^2.3.0",
-    "stylelint-scss": "^3.18.0",
     "swr": "^0.5.6",
     "use-resize-observer": "^6.1.0"
+  },
+  "devDependencies": {
```
Comment (alexgoff) [inline package.json:72]: relocated dev dependencies, consulted package pages to confirm if they are intended for dev use only

File: package.json [hunk 2]
```diff
@@ -83,10 +66,33 @@
...
+    "eslint-plugin-react-hooks": "^4.2.0",
+    "json-to-scss": "^1.6.2",
+    "npm-run-all": "^4.1.5",
+    "postcss-normalize": "^10.0.1",
+    "postcss-preset-env": "^6.7.0",
+    "stylelint": "^13.8.0",
+    "stylelint-config-recess-order": "^2.3.0",
+    "stylelint-scss": "^3.18.0"
+  },
+  "resolutions": {
```
Comment (alexgoff) [inline package.json:94]: resolutions to reach the dependency packages that were not updated by updating their parents

File: package.json [hunk 3]
```diff
@@ -83,10 +66,33 @@
...
+    "eslint-plugin-import": "^2.22.1",
+    "eslint-plugin-jsx-a11y": "^6.4.1",
+    "eslint-plugin-node": "^11.1.0",
+    "eslint-plugin-prettier": "^3.1.4",
+    "eslint-plugin-promise": "^4.2.1",
+    "eslint-plugin-react": "^7.21.5",
+    "eslint-plugin-react-hooks": "^4.2.0",
+    "json-to-scss": "^1.6.2",
+    "npm-run-all": "^4.1.5",
+    "postcss-normalize": "^10.0.1",
```
Comment (alexgoff) [inline package.json:88]: `postcss-normalize` updated from `9.0.0` to `10.0.1` to resolve a dependency issue after `sanitize.css` was updated when the `yarn.lock` file was refreshed

File: pages/tours/_styles.scss [hunk 4]
```diff
@@ -18,7 +19,7 @@
 
 .tour-summary-container {
   $summaryLinkNumberSize: 34px;
-  $summarySpacing: $summaryLinkNumberSize / 2;
+  $summarySpacing: math.div($summaryLinkNumberSize,2);
```
[the remark this answers is not in the document]
Comment (blnkt) [reply pages/tours/_styles.scss:22]: good catch

**Review by blnkt — APPROVED** [review]

**Pull request merged**